src/main/java/seedu/address/model/person/Person.java

```diff
@@ -163,11 +173,25 @@ public String toString() {
         return builder.toString();
     }
 
+    /**
+     * Returns true if a person is a doctor.
+     * Returns false otherwise.
+     */
     public boolean isDoctor() {
```

Comment:
Since we already have this method (and isPatient), is there still a need to override this in the Doctor and Patient classes?